### ISSUE TITLE
Floating error - secp256k1N.sub(...).toBuffer is not a function

### ISSUE THREAD

**Issue by VanijaDev** [odanado/aws-kms-provider#486]
https://github.com/odanado/aws-kms-provider/blob/f686cfd347c6910c1993a709057889686306efbb/aws-kms-packages/aws-kms-signer/src/signature.ts#L31

Hello, I am using this package while AWS KMS implementation. It works great most of time, but sometimes I get error "TypeError: secp256k1N.sub(...).toBuffer is not a function". I investigated a bit and found, that it occurs in this place. The error is floating because sometimes I get this error 3 times in a row, sometimes I get successful tx 5 times in a row. It may fail on first transaction, may fail after 10sec, 1 minute.

The same floating error while using eth.sendTransaction as in example.
```
const receipt = await web3.eth.sendTransaction({
      from: FROM_ADDRESS,
      _receiver
    });

```

Please see screenshots attached for more info. Looking forward hearing from you & willing to assist to resolve it.

<img width="1918" alt="Screenshot 2022-03-25 at 17 19 30" src="https://user-images.githubusercontent.com/15075510/160210891-a48cdf54-c8ec-4f3e-81d2-e3ada342475a.png">
<img width="1097" alt="Screenshot 2022-03-25 at 17 33 03" src="https://user-images.githubusercontent.com/15075510/160210895-af9580d4-fbcb-4433-a2c2-06e119c71cb6.png">

**Comment by odanado** [comment]
@VanijaDev Thank you for a reporting.

From that screenshot it looks like you are running aws-kms-provider in your browser. So I have a few questions.

- What is your browser type and version?
- What is the type and version of Web Framework?
- In which case does this error occur, SSR or CSR?

**Comment by VanijaDev** [comment]
Hi,
- Google Chrome Version 99.0.4844.83 (Official Build) (x86_64)

- I don’t use any frameworks. HTML + JS, it’s for testing only

**Comment by odanado** [comment]
Thanks for the additional info!

Maybe, this bug is related to https://github.com/indutny/bn.js/issues/263.
I will release a version with the bug fixed in a few days.

**Comment by odanado** [comment]
@VanijaDev 
I fixed this issue at aws-kms-provider@0.6.0
